### PR TITLE
Feature/allow reference of existing k8s secrets

### PR DESCRIPTION
**Pull request by crimbler2**
Title is misleading. Chart now dynamically references secrets that are created by crossplane. There needs to be a convention about the name of course

### CODE REVIEW

File: charts/dvpe-deployment-gloo/templates/deployment.yaml [hunk 1]
```diff
@@ -86,10 +86,21 @@ spec:
           {{- if .Values.additionalparameters.secretsApplied }}
           {{- range $key, $value := .Values.additionalparameters.secrets }}
           - name: "{{$key}}"
+            valueFrom:
+              secretKeyRef: {{ $value.secretKeyRef }}
+          {{- end}}
+          {{- end}}
+            {{- if .Values.infrastructure }}
```
Comment (christodo) [inline charts/dvpe-deployment-gloo/templates/deployment.yaml:93]: you need to add your values to the values.yaml. Otherwise it is not documented

**Review by christodo — CHANGES_REQUESTED** [review]

File: charts/dvpe-deployment-gloo/templates/deployment.yaml (hunk 1 above)
Comment (crimbler2) [reply charts/dvpe-deployment-gloo/templates/deployment.yaml:93]: Only those used in this github repo or also those used by the actual templates?

**Review by christodo — CHANGES_REQUESTED** [review]
As discussed probably not the right way to do it

**Review by mithie — APPROVED** [review]
perfect! as simple as can be

**Review by christodo — APPROVED** [review]

**Pull request merged**